src/libAtomVM/term.h

```diff
@@ -1313,14 +1313,14 @@ static inline int term_list_length(term t, int *proper)
 }
 
 /**
- * @brief Returns 1 if given terms are exactly equal.
+ * @brief Returns \c true if given terms are exactly equal.
  *
- * @details Compares 2 given terms and returns 1 if they are the same.
+ * @details Compares 2 given terms and returns true if they are the same.
  * @param a first term
  * @param b second term
- * @return 1 if they are the same, 0 otherwise.
+ * @return \c true if they are the same, \c false otherwise.
  */
-static inline int term_exactly_equals(term a, term b, Context *ctx)
+static inline bool term_exactly_equals(term a, term b, Context *ctx)
 {
     if (a == b) {
         return 1;
```

Comment:
We are missing a true here.
And optionally this can be simplified with an or expression.